lectures/networks.md

```diff
@@ -1177,10 +1177,9 @@ Japan takes the highest rank according to this measure, although
 countries with large financial sectors such as Great Britain and France are
 not far behind.
 
-The advantage of eigenvector centrality is that it measures a node’s importance while giving consideration
-to the importance of its neighbors.
+The advantage of eigenvector centrality is that it measures a node’s importance while considering the importance of its neighbours.
```

Comment:
Is that a non ascii character in "node's"  And maybe the same elsewhere?

Reply:
Also found one below and fixed that as well.